platform/viewer/cypress/integration/study-list/OHIFStudyList.spec.js

```diff
@@ -21,31 +21,42 @@ describe('OHIF Study List', function() {
       //cy.get('@modalities')
     });
 
-    it('searches Patient Name with exact string', function() {
+    it('Displays several studies initially', function () {
```

Comment:
There wasn't an initial check previously to ensure that the base state was correct.  This check will fail in previous versions on the length > 1

Reply:
nice